backend/src/ee/services/permission/permission-dal.ts

```diff
@@ -69,9 +74,22 @@ export const permissionDALFactory = (db: TDbClient) => {
           db.ref("updatedAt").withSchema(TableName.ProjectMembership).as("membershipUpdatedAt"),
           db.ref("authEnforced").withSchema(TableName.Organization).as("orgAuthEnforced"),
           db.ref("orgId").withSchema(TableName.Project),
-          db.ref("slug").withSchema(TableName.ProjectRoles).as("customRoleSlug")
-        )
-        .select("permissions");
+          db.ref("slug").withSchema(TableName.ProjectRoles).as("customRoleSlug"),
+          db.ref("permissions").withSchema(TableName.ProjectRoles),
+          db.ref("id").withSchema(TableName.ProjectUserAdditionalPrivilege).as("userApId"),
```

Comment:
general nit: if you can use the long form instead of the abbreviation `ap` that would increase readability

Reply:
The ap is short form because it is used as an alias and immediately changed to the right one in the next one.

Reply:
yeah i can see that it gets renamed in later steps but it helps to quickly read because for a second i was like what is `ap` and i think others will think the same

Reply:
Hm will keep in mind. But its generally i didn't see the value as it gets immediately changed